### DISCUSSION TITLE
Systems statistics meaning

### DISCUSSION THREAD

**Discussion by Steven630** [koreader/koreader#12679] [Q&A] [answered]
I have two 6-inch ebook devices. One is 2+32GB, with better processor (eight cores). The other has 4+32, though with slower four-core CPU .

With 2+32:

Virtual memory 15342.52
Ram usage 179.62
Total memory 1632.36
Free memory 131.73
Available memory 214.88

With 4+32:

Virtual memory 5193.91
Ram usage 273.55
Total memory 3838.25
Free memory  212.73
Available memory 2679.11

I'm having difficulty understanding all of these.  What's the difference between free memory and available memory?  The two ereaders both have a storage of 32GB. Why do the statistics differ so much? ( The seconds's available memory is ten times that of the first, while the first has three times more virtual memory.)

**Comment by Frenzie** [comment, marked as answer]
Free memory isn't doing anything at all. It's considered wasted. Available memory is storing a variety of things which can be tossed in principle.

https://man7.org/linux/man-pages/man1/free.1.html
